homeassistant/components/command_line/binary_sensor.py

```diff
@@ -98,6 +104,7 @@ async def async_setup_platform(
                 payload_off,
                 value_template,
                 unique_id,
+                scan_interval,
             )
         ],
         True,
```

Comment:
We don't need to pass `True` here anymore with the new code.

Reply:
Fixed in https://github.com/home-assistant/core/pull/94040